libs/MeshKernel/src/CurvilinearGrid/CurvilinearGrid.cpp

```diff
@@ -158,24 +158,6 @@ void CurvilinearGrid::Delete(std::shared_ptr<Polygons> polygons, UInt polygonInd
     }
 }
 
-bool CurvilinearGrid::IsValid() const
-{
-    if (m_gridNodes.size() == 0)
-    {
-        return false;
-    }
-    if (NumM() < 2)
-    {
-        return false;
-    }
-    if (NumN() < 2)
-    {
-        return false;
-    }
-
-    return true;
-}
-
 void CurvilinearGrid::BuildTree(Location location)
 {
```

Comment:
Couldn't this be implemented as a call to the override with bounding box, passing in a default bounding box?